.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: User Story, To be verify
+---
+
+## Description
+
+**As a** `[roles impacted]`,
+
+**I want** `[clear context and goals]`
+
+**so that** `[result of this story]`.
+
+## Acceptance Criteria
+
+ **Scenario** | `[the name for the behavior that will be described]`
+ ------------ | -------------------------------------------------
+ **Given**    | `[the beginning state of the scenario]`
+ **When**     | `[specific action that the user makes]`
+ **Then**     | `[the outcome of the action in “When”]`
+ **And**      | `[used to continue any of three previous statements]`
+
+### Constraints
+
+`[Put all others constraints here, like list of acceptances values or other]`
```

Comment:
acceptance values

Reply:
no modification, I will update all other issue template in another task.